docs/_snippets/architecture/blocs_presentation.dart.md

```diff
@@ -4,18 +4,18 @@ class MyWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocListener<WeatherCubit, WeatherState>(
+    return BlocListener<WeatherBloc, WeatherState>(
```

Comment:
If we make this change we will also need to update the corresponding documentation in https://github.com/felangel/bloc/blob/b11fe6a1c00a5cb22fd38b2eba2ae6c0c773cec5/docs/architecture.md?plain=1#L73 (and other translations). The simpler change would be to go back to using `FirstBloc` and `SecondBloc` in order to be consistent with the rest of the docs. Let me know what you think and thanks for the contribution! 🙏